deploy/restartComponent.sh

```diff
@@ -1,18 +1,22 @@
 #!/bin/sh
+## Pass the component names as command line arguments, e.g.:
+## ./restartComponent.sh ErrorHandler JobSubmitter AgentStatusWatcher
 
 HOST=`hostname`
 DATENOW=`date +%s`
 
+# Get a few environment variables in, like $install and $manage
+source /data/admin/wmagent/env.sh
+
 # Figure whether it's a python2 or python3 agent
-if [ -d "/data/srv/wmagent/current/install/wmagent" ]; then
-  WMA_PATH="/data/srv/wmagent/current/install/wmagent"
-else
-  WMA_PATH="/data/srv/wmagent/current/install/wmagentpy3"
+if [ ! -d "$install" ]; then
+  install="/data/srv/wmagent/current/install/wmagentpy3"
```

Comment:
I know the following component does not relate to this exact script (so no action expected - just a comment).
I think python3 is the de-facto only version we are going to support from now on... and we never plan to move back to py2 or to prolong any support of it.So we should think of getting rid of this  `py3` suffix.

Reply:
Yes, I thought the same as well. The positive aspect of keeping this logic around is that it will ease the migration process to a new major python release. However, there is no release date for python4 neither certainty that it will be ever released.